scipy/spatial/transform/rotation.pyx

```diff
@@ -878,22 +878,27 @@ cdef class Rotation(object):
         Parameters
         ----------
         seq : string
-            Specifies sequence of axes for rotations. Up to 3 characters
+            Specifies the order of the sequence of
```

Comment:
I don't see that this change makes the description convincingly more clear.

"the order of the sequence"  Can it be reduce to "the sequence"?

Also you removed "axes" completely, but "axes" are very relevant here. So if we remove "the order" and return "axes" we'll get back to the current text.

Reply:
The term "sequence", in my opinion, does not explicitly indicate that the argument is meant to indicate both the the order of rotations and the axes about which they are performed. For this reason I find the existing text exceedingly unclear. 

But I think it might be ok if we do a better job at explicitly defining the `angles` argument. Let me pick up the conversation in your next comment.